paritytech/parity-signer
# feat: support pinless signing when user already input pin recently

closes #489 

This PR based on #565 , would be easier to view that one first.

### Pinlos actions

When user input pin once for a identity, pin will not needed for the following actions of this identity:
* new account derivatiopn.
* signing request for acccount without extra password.
* signing request for passworded account only need to input password.

remove pin requirement when deleting account, but when delete identity, the pin will be always required.

### How to test

Need to run `yarn start --reset-cache` when start react-native server.

tested cases in e2e:
* move app to background and come back again, all data pointer should be cleared.
* when creating new identity, following actions do not need pin any more
* for passworded account, once seed reference existed, next time only password is needed for signing.
* delete identity always need pin.
